apps/api/src/rest-app.ts

```diff
@@ -89,8 +86,6 @@ appHono.use(
   })
 );
 
-const { PORT = 3080 } = process.env;
-
 const scheduler = new Scheduler({
```

Comment:
The scheduler isn't used in api, we can safely removed it

Reply:
I decided to remove in the next PRs